keyboards/eyeohdesigns/s50/s50.h

```diff
@@ -0,0 +1,238 @@
+/* Copyright 2020 Eye Oh Designs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+//#pragma once
+#ifndef S50_H
+#define S50_H
```

Comment:
```suggestion
#pragma once

```

Reply:
I've made all the other changes, but making this change breaks the code and will not allow me to compile.

Reply:
Got it fixed and committed.